test_scripts/RC/OnRCStatus/015_notification_by_user_exit.lua

```diff
@@ -0,0 +1,57 @@
+---------------------------------------------------------------------------------------------------
+-- User story: TBD
+-- Use case: TBD
+--
+-- Requirement summary:
+-- [SDL_RC] TBD
+--
+-- Description: SDL shall send OnRCStatus notifications to rc registered app
+-- by allocation module via GetInteriorVehicleData
```

Comment:
@GetmanetsIrina Please update description that is reflect the code

Reply:
Locally changed.